decision_making.md

```diff
@@ -2,40 +2,45 @@
-  - The safeguards of (1) publicly notifying all decision makers that a vote has been called and (2) requiring a 7-day voting period to elapse protect against unilateral decision-making.
 - Special guidelines for software projects making decisions:
   - Add a link to this document in your issue and PR templates along with a link to the decision making body membership (see recommended language below).
   - Create a “Consensus Seeking” and “Vote” tags to make it easy to find issues/PRs where a decision is being made.
   - When a vote is called on GitHub, the sponsor should summarize the decision and paste a checklist of the voting members to use in voting in the top entry (description) of the issue/PR.
   - When communications (in whatever number of channels) point to a discussion that requires a project-wide decision, and before major implementation work starts, a standalone issue should be opened that summarizes the issue, points to relevant prior discussions, and calls for an explicit vote/decision. This issue should be tightly scoped to the relevant decision only, and include the "Vote" tag.
   - When an implementation of a decision appears in pull request, link to the relevant “Vote” issue.
-- Decision making bodies and those proposing decisions should explicitly distinguish between decisions that are two-way doors (easy to reverse later) and one-way doors (difficult or impossible to reverse later). For one-way doors, teams should carefully weigh alternatives and tradeoffs and take extra care to ensure broad participation and stakeholder input. For two-way doors, teams should feel free to move quickly, without compromising the principles and procedures described herein. 
-- Do not call a vote to short-circuit an ongoing discussion that is still productive in terms of exploring ideas and feedback. Votes should be called only when discussion has reasonably explored the space and stakeholders have had a voice, to clarify if lack of input is consensus or simply folks being busy, or to record key decisions where an explicit vote will be useful (e.g. to signal an important decision to the broader community).
+- Decision making bodies and those proposing decisions should explicitly distinguish between decisions that are two-way doors (easy to reverse later) and one-way doors (difficult or impossible to reverse later). For one-way doors, teams should carefully weigh alternatives and tradeoffs and take extra care to ensure broad participation and stakeholder input. For two-way doors, teams should feel free to move quickly, without compromising the principles and procedures described herein.
+
+reasonably explored the space and stakeholders have had a voice, to clarify if lack of input is consensus or simply folks being busy, or to record key decisions where an explicit vote will be useful (e.g. to signal an important decision to the broader community).
```

Comment:
I think this line was leftover from a previous edit. The gist is captured on lines 32-33, so I suggest removing this, or incorporating it on 32-33 above.